app/modules/power/README.md

```diff
@@ -1,24 +1,99 @@
-Power module
+Power Module
 ==============
 
-Reports on Power information 
-
-- Battery information
-Note: Battery condition can fluctuate between "Service Battery" and "Normal".
+Reports on Power and Battery information 
 
 - Power Supply information (Energy Saving settings)
 - Sleep / Wake settings
-
+- Other Power related information from pmset
+- Battery information
+Note: Battery condition can fluctuate between "Service Battery" and "Normal".
 
 Data can be viewed at
 - Power under Listings menu 
 - Power tab on the client details page
 - Power widgets under Reports menu
+- Batteries under Listing menu
+- Battery tab on the client details page
 
+Power module
+==============
 
-
-TO DO
----
-
-*** Power Supply information
-*** Sleep / Wake settings
+* manufacture_date - VARCHAR(255) - Date of battery's manufacture
+* design_capacity - INT(11) - Design capacity of battery
+* max_capacity - INT(11) - Maximum capacity of battery
+* max_percent - INT(11) - Maximum percent of battery
+* current_capacity - INT(11) - Current capacity of battery
+* current_percent - INT(11) - Current percent of battery
+* cycle_count - INT(11) - Amount of battery cycles
+* temperature - INT(11) - Temperature of battery
+* condition - VARCHAR(255) - Current battery condition
+* timestamp - INT(11) - When data was las uploaded
+* sleep_prevented_by - TEXT - List of processes preventing sleep
+* hibernatefile - VARCHAR(255) - Path of hibernate file
+* schedule - VARCHAR(255) - Scheduled events
+* adapter_id - VARCHAR(255) - ID of power adapter
+* family_code - VARCHAR(255) - Family code of adapter
+* adapter_serial_number - VARCHAR(255) - Serial number of adapter
+* combined_sys_load - VARCHAR(255) - Combined system load status
+* user_sys_load - VARCHAR(255) - User system load status
+* thermal_level - VARCHAR(255) - Therm level status
+* battery_level - VARCHAR(255) - Battery level status
+* ups_name - VARCHAR(255) - Name of UPS
+* active_profile - VARCHAR(255) - Currently active power profile
+* ups_charging_status - VARCHAR(255) - Charging status of UPS
+* externalconnected - VARCHAR(255) - Is power adapter connected
+* cellvoltage - VARCHAR(255) - Voltage levels of battery cells
+* manufacturer - VARCHAR(255) - Manufacturer of battery
+* batteryserialnumber - VARCHAR(255) - Serial number of battery
+* fullycharged - VARCHAR(255) - Is battery fully charged
+* ischarging - VARCHAR(255) - Is battery charging
+* standbydelay - INT(11) - Minutes until standby
```

Comment:
According to the manage, standby delay is in seconds: "standbydelay specifies the delay, in seconds, before writing the hibernation image to disk and powering off memory for Standby."